packages/exemplar-storybook-react/package.json

```
@@ -50,7 +50,7 @@
     "@babel/core": "^7.4.5",
     "@babel/plugin-syntax-dynamic-import": "^7.2.0",
     "@storybook/addons": "^5.1.9",
-    "@storybook/react": "^5.1.0",
+    "@storybook/react": "5.2.8",
```

do you want this instead?
```suggestion
    "@storybook/react": "~5.2.8",
```